savelearner.py

```diff
@@ -1,19 +1,19 @@
-"""Importo learner un file che ho già creato in precedenza"""
+# Importo learner un file che ho già creato in precedenza
```

Comment:
""" sono docstrings e vengono usate per la documentazione, ovvero la parte di spiegazione esposta direttamente all'utente finale. I commenti sono invece più "personali", anche se è evidente che se metti il codice su GitHub in teoria chiunque può vederlo.

Ad ogni modo: **docstrings** per spiegare cosa fa e come si usa un blocco di codice (classe, metodo o funzione); **#** per ricordarti perché o come hai fatto qualcosa